notebooks/official/generative_ai/text_embedding.ipynb

```diff
@@ -0,0 +1,1825 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        get_embedding.counter += 1
</pre><p>Storing state in a function like this is technically possible in Python but I wouldn't say it's common. Consider removing state and just sleeping for the rate-limit (i.e. time.sleep(1/QPS)).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>